src/Suave/Tcp.fs

```diff
@@ -140,6 +140,7 @@ let runServer maxConcurrentOps bufferSize autoGrow (binding: SocketBinding) star
               (acceptingConnections: AsyncResultCell<StartedData>) serveClient = async {
   try
     use listenSocket = new Socket(binding.endpoint.AddressFamily, SocketType.Stream, ProtocolType.Tcp)
+    listenSocket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.ReuseAddress, 1)
```

Comment:
here i would actuall add ```#if NETSTANDARD2_0``` - to explicitly denote that this code is relevant only for NetCore, because there is not need of it for full-framework